settlement/tests/treasury.rs

```diff
@@ -182,6 +182,7 @@ fn relay_1() {
         author: reserved_state.query_name(&keys[0].0).unwrap(),
         timestamp: 0,
         transactions_hash: Agenda::calculate_transactions_hash(&[tx1.clone(), tx2.clone()]),
+        previous_block_hash: genesis_header.to_hash256(),
```

Comment:
Ditto.

Reply:
reflected

Reply:
Not fixed.

Reply:
sorry for the misunderstanding. reflected.